### PR TITLE
Add overlay sample

### PR DESCRIPTION
**Pull request by atanasiuk-hubspot**
Shows ways of using `Panel` and `Modal` components.

image

### CODE REVIEW

File: overlay-example/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Quick Start
+
+### Step 1: Update your CLI and & authenticate your account
+
+1. Update to latest CLI version by running `npm install -g @hubspot/cli@latest`.
+1. Run `hs init` if you haven’t already done so to create a config file for your parent account.
+1. Run `hs auth` to authenticate your account. Alternatively, select your pre-authenticated account with `hs accounts use`.
+
+### Step 2: Create Flex and Box project
```
Comment (aanchalHS) [inline overlay-example/README.md:13]: I will edit these once we merge

File: config.json [hunk 2]
```diff
@@ -59,6 +59,12 @@
       "label": "Custom Logger Example",
```
Comment (aanchalHS) [inline config.json:59]: This might need to be changed

File: config.json [hunk 3]
```diff
@@ -59,6 +59,12 @@
...
       "path": "custom-logger-example",
```
Comment (aanchalHS) [inline config.json:60]: same here

**Review by aanchalHS — COMMENTED** [review]
Is it possible for us to change this?

File: config.json (hunk 3 above)
Comment (atanasiuk-hubspot) [reply config.json:60]: What is wrong with custom-logger sample?

**Review by skaplan-dev — APPROVED** [review]

**Pull request merged**